roles/custom/matrix-bridge-zulip/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2021 Toni Spets
+# SPDX-FileCopyrightText: 2022 - 2024 Slavi Pantaleev
+# SPDX-FileCopyrightText: 2022 Marko Weltzer
+# SPDX-FileCopyrightText: 2025 Suguru Hirahara
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+---
+- name: Perform MatrixZulipBridge installation tasks
```

Comment:
Please use `Zulip bridge` as name in docs and tasks - readability is the key

Reply:
Addressed with https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/4695/commits/bcce68da1f5dd034356e20c7f0d4aed2be301c57.